docs/hosting/deploy.md

```diff
@@ -1,20 +1,417 @@
+# Reflex Hosting Service
+
 ```python exec
 import reflex as rx
+from pcweb import constants
+from pcweb.templates.docpage import doccmdoutput
+from reflex.components.radix.themes.components import *
+from reflex.components.radix.themes.layout import *
+from reflex.components.radix.themes.typography import *
 ```
 
-# Reflex Hosting Service
+So far, we have been running our apps locally on our own machines.
+But what if we want to share our apps with the world?  This is where
+the hosting service comes in.
 
 ```md alert info
-# Hosting is in Alpha
-Please reach out to us on Discord if you have an app ready to deploy and we will give you an invitation code.
+Hosting is in Alpha. Please reach out to us on Discord if you are ready to deploy and we will give you an invitation code.
 ```
 
-So far, we've been running our apps locally on our own machines.
-But what if we want to share our apps with the world?  This is where
-deployment comes in.
+## Quick Start
+
+Reflex’s hosting service makes it easy to deploy your apps without worrying about configuring the infrastructure.
+
+```md alert info
+This tutorial assumes you have successfully `reflex init` and `reflex run` your app.
+```
+
+### Authentication
+
+```md alert info
+Hosting service requires `reflex>=0.3.2`.
+```
+
+First, create an account or log into it using the following command.
+
+```bash
+reflex login
+```
+
+You will be redirected to your browser where you can authenticate through Github or Gmail.
+
+### Deployment
+
+Once you have successfully authenticated, you can start deploying your apps.
+
+```md alert warning
+Make sure you have a `requirements.txt`  file at the top level app directory that contains all your python dependencies!
+```
```

Comment:
Lots of alerts here its a little overwhelming maybe have a section called pre requisites which tells them what they need before deploying

Reply:
sounds good. move the appropriate alerts to a prereq section